src/crawler/crawl.ts

```diff
@@ -47,6 +50,51 @@ class Crawler {
     return encodeNodePublic(Buffer.from(publicKey, 'base64'))
   }
 
+  /**
+   * Helper function for determining if a node's newest ledger is close to the current network's range.
+   *
+   * @param thisCompleteLedgers - The `complete_ledgers` for the current node.
+   * @param nodeCompleteLedgers - The `complete_ledgers` for the new node.
+   * @returns Whether the network is in a valid range.
+   */
+  private static ledgerInRange(
+    thisCompleteLedgers: string | undefined,
+    nodeCompleteLedgers: string | undefined,
+  ): boolean {
+    const newestLedger = Crawler.getRecentLedger(thisCompleteLedgers)
+    const nodeNewestLedger = Crawler.getRecentLedger(nodeCompleteLedgers)
+    if (newestLedger == null || nodeNewestLedger == null) {
+      return false
+    }
+
+    const intNewestLedger = parseInt(newestLedger, 10)
+    const intNodeNewestLedger = parseInt(nodeNewestLedger, 10)
+    if (
```

Comment:
IMO this is more readable. If I read this correctly this is just checking if we're in range.

```js
Math.abs(intNewestLedger - intNodeNewestLedger) < LEDGER_RANGE
```